src/plugins/minimal_scene/MinimalScene.cc

```diff
@@ -841,6 +843,7 @@ void RenderWindowItem::keyReleaseEvent(QKeyEvent *_e)
 void RenderWindowItem::mouseReleaseEvent(QMouseEvent *_e)
 {
   this->dataPtr->mouseEvent = convert(*_e);
+  this->dataPtr->mouseEvent.SetPressPos(_e->pos().x(), _e->pos().y());
```

Comment:
This event is `mouseRelease`, not `mousePress`, so I think this may be misplaced

Reply:
There is only one mouseEvent, it's used for press, hover and release. This is required to know where was the mouse released.

Reply:
But we could also populate it on `RenderWindowItem::mousePressEvent`, right?

Reply:
It's done 

https://github.com/ignitionrobotics/ign-gui/blob/bde1977b4600602fb63f08d74cfbeeaf5bd3919b/src/plugins/minimal_scene/MinimalScene.cc#L812-L815

Reply:
Ah ok, so what I don't understand is why we'd set `PressPos` during the mouse release. AFAICT, we store the press position so that we can later compare it to the release position.

What's probably happening is that we're losing the original press position when we convert, right? Maybe we should store the press pos before convert and then set that. i.e.

```cpp
auto pressPos = this->dataPtr->mouseEvent.PressPos();
this->dataPtr->mouseEvent = convert(*_e);
this->dataPtr->mouseEvent.SetPressPos(pressPos);
```